.rtd-env-py3.yml

```diff
@@ -6,7 +6,11 @@ dependencies:
 - python=3.8
 - mpi4py
 - islpy
+- pip
 - pip:
+  - "pybind11"
+  - "mako"
+  - "pyopencl"
```

Comment:
Why get pyopencl from pip (where it might need compilation)? (rather than conda, where it doesn't)

Reply:
Ok, that fixed it.